velox/docs/functions/spark/datetime.rst

```diff
@@ -7,6 +7,13 @@ Convenience Extraction Functions
 
 These functions support TIMESTAMP and DATE input types.
 
+.. spark:function:: add_months(startDate, numMonths) -> integer
+
+    Returns the date that is ``numMonths`` after ``startDate``.
+    When ``numMonths`` is zero returns ``startDate`` unmodified.
```

Comment:
It would be nice to add some examples, especially for corner cases like adding or subtracting 12 months to Feb 29 or add 1 month to May 31.